spanner/api/Spanner/CreateInstance.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_create_instance]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Spanner.Admin.Instance.V1;
+using Google.Cloud.Spanner.Common.V1;
+using Google.LongRunning;
+using System;
+
+public class CreateInstance
+{
+    public static Instance SpannerCreateInstance(string projectId, string instanceId)
+    {
+        // Create the InstanceAdminClient instance.
+        InstanceAdminClient instanceAdminClient = InstanceAdminClient.Create();
+
+        // Initialize request parameters.
+        Instance instance = new Instance
+        {
+            InstanceName = InstanceName.FromProjectInstance(projectId, instanceId),
+            ConfigAsInstanceConfigName =
+                InstanceConfigName.FromProjectInstanceConfig(projectId, "regional-us-central1"),
```

Comment:
Just a comment: When we do the rewrite we would ideally remove all the command line runner bits. At that point I belive we should move away from attempting command line related consistency with samples in other languages. Are you OK with that?

Reply:
Yeah I think that should be fine. It's just from the perspective of the docs team, they try to write one description of what the sample is doing and ideally the description would apply consistently to all 8 clients. But some small minor variations should be fine as long as main thing the sample is trying to show is consistent.